Added index.html -closes issue#1-"Create an index file for my website"

SefisoMonama opened the pull request.

SefisoMonama committed:
Added index.html -closes issue#1-"Create an index file for my website"

dabarnard commented:
@SefisoMonama , well done. This looks good. I just want to show you one tip wrt Github, you can refer to issues/PRs by prefixing it with a # . So you will see, right here, if I write #2 it will refer to this pull request. Try it in your description field of your PR - mention that this PR closes issue #1 , then you will see that once you accept the PR, the issue will automatically be closed.

This is the typical workflow we use:

- You create various issues which indicate bugs, features or any changes which need to be made.

- You then create a branch and work on these issues.

- You then create a PR which addresses these issues (mention them in the PR with the # ). 

- Once the PR is approved, the issues automatically move to closed.

This makes it easy to see which items are outstanding vs which ones are fixed.Let me know if this doesn't make sense or if you have any questions.

SefisoMonama commented:
I understand the logic behind this Dieter, I take it works just like @-
mention, you use it to call a specific PR or issue

On Tue, 10 Nov 2020, 09:00 dabarnard, <notifications@github.com> wrote:

> @SefisoMonama <https://github.com/SefisoMonama> , well done. This looks
> good. I just want to show you one tip wrt Github, you can refer to
> issues/PRs by prefixing it with a # . So you will see, right here, if I
> write #2 <https://github.com/SefisoMonama/Resume/pull/2> it will refer to
> this pull request. Try it in your description field of your PR - mention
> that this PR closes issue #1
> <https://github.com/SefisoMonama/Resume/issues/1> , then you will see
> that once you accept the PR, the issue will automatically be closed.
>
> This is the typical workflow we use:
>
>    -
>
>    You create various issues which indicate bugs, features or any changes
>    which need to be made.
>    -
>
>    You then create a branch and work on these issues.
>    -
>
>    You then create a PR which addresses these issues (mention them in the
>    PR with the # ).
>    -
>
>    Once the PR is approved, the issues automatically move to closed.
>
> This makes it easy to see which items are outstanding vs which ones are
> fixed.Let me know if this doesn't make sense or if you have any questions.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/SefisoMonama/Resume/pull/2#issuecomment-724505403>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARRHP2XR4VWDWGSCC5JHOZLSPDQHTANCNFSM4TQICIIQ>
> .
>

dabarnard commented:
Exactly. Go ahead and accept the PR
